serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/GlueSchemaRegistrySerializationFacade.java

```diff
@@ -65,8 +64,7 @@ public GlueSchemaRegistrySerializationFacade(@NonNull AwsCredentialsProvider cre
 
         if (this.schemaByDefinitionFetcher == null) {
             final AWSSchemaRegistryClient client =
-                new AWSSchemaRegistryClient(credentialProvider, this.glueSchemaRegistryConfiguration,
-                    AWSSchemaRegistryGlueClientRetryPolicyHelper.getRetryPolicy());
```

Comment:
retry policy will get automatically filled in by the less-telescoped constructor